Gems/RecastNavigation/Code/Tests/NavigationMeshTest.cpp

```diff
@@ -704,6 +704,8 @@ namespace RecastNavigationTests
         wait.BlockUntilCalled();
     }
 
+    // Disabling this test to unblock AR while an investigation is in progress.
+    /*
     TEST_F(NavigationTest, AsyncRerun)
```

Comment:
A better way to disable a unit test is to add `DISABLED_` to its name:
```
TEST_F(NavigationTest, DISABLED_AsyncRerun)
```
As per the GoogleTest docs: https://github.com/google/googletest/blob/main/docs/advanced.md#temporarily-disabling-tests

Reply:
Done